pkg/controllers/scheduler/framework/plugins/clusterresources/most_allocated.go

```diff
@@ -49,19 +49,25 @@ func (pl *ClusterResourcesMostAllocated) Score(
 		return 0, framework.NewResult(framework.Error, err.Error())
 	}
 
-	requested := make(framework.ResourceToValueMap, len(framework.DefaultRequestedRatioResources))
-	allocatable := make(framework.ResourceToValueMap, len(framework.DefaultRequestedRatioResources))
-	for resource := range framework.DefaultRequestedRatioResources {
+	resources := framework.GetRelevantResources(su)
```

Comment:
There are the same logic for getting variables like resources, requested and allocatable in balanced/least/most_allocated.go files.Can u extract them as a utility function?

Reply:
Done